app/src/templates/workspace.html

```diff
@@ -16,7 +16,7 @@
             <div class="form-group">
               <label for="collectionTitleForm">Collection</label>
               <div class="input-group">
-                <div class="form-control" style="min-width: 300px" editable-text="wc.collection.title" e-form="collectionTitleForm" e-name="collectionTitleForm" e-maxlength="255" onaftersave="wc.saveCollection()" ng-click="collectionTitleForm.$show()">
+                <div class="form-control" style="min-width: 200px" editable-text="wc.collection.title" e-form="collectionTitleForm" e-name="collectionTitleForm" e-maxlength="255" onaftersave="wc.saveCollection()" ng-click="collectionTitleForm.$show()">
```

Comment:
Changed this so that when the library is open, both the editable title field and edit description button can fit on one line.